CONTRIBUTING.md

```diff
@@ -98,15 +98,30 @@ $ git commit
 Writing good commit logs is important. A commit log should describe what
 changed and why. Follow these guidelines when writing one:
 
-1. The first line should be 50 characters or less and contain a short
-   description of the change. All words in the description should be in
-   lowercase with the exception of proper nouns, acronyms, and the ones that
-   refer to code, like function/variable names. The description should
-   be prefixed with the name of the changed subsystem and start with an
-   imperative verb. Example: "net: add localAddress and localPort
-   to Socket"
+#### Commit Message Guidelines
```

Comment:
I was wondering if you think this should make a distinction between the pr title and description vs. the various commits that might make up a PR, including ones from feedback.